auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Routing.kt

```diff
@@ -66,11 +72,37 @@ fun Route.externalRoutes(
         )
     }
 
+    route("/delta/register") {
+        deltaRegisterRoutes(deltaUserRegistrationController)
+    }
+
+    route("/delta/set-password"){
+        deltaSetPasswordRoutes(deltaSetPasswordController)
+    }
+
     route("/delta") {
         deltaLoginRoutes(serviceConfig, deltaLoginController, deltaSSOLoginController)
     }
 }
 
+fun Route.deltaSetPasswordRoutes(deltaSetPasswordController: DeltaSetPasswordController) {
+    route("/success") {
+        deltaSetPasswordController.setPasswordSuccessRoute(this)
+    }
+    // TODO - should this be rate limited?
+    deltaSetPasswordController.setPasswordFormRoutes(this)
```

Comment:
Wouldn't hurt, though there shouldn't be any practical attacks on it. The register endpoint definitely should be though.